Client/Frontend/Login Management/LoginListTableViewCell.swift

```diff
@@ -8,61 +8,60 @@ class LoginListTableViewSettingsCell: ThemedTableViewCell {
     override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
         super.init(style: style, reuseIdentifier: reuseIdentifier)
     }
-    
+
     required init?(coder aDecoder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
 }
 
 class LoginListTableViewCell: ThemedTableViewCell {
     private let breachAlertSize: CGFloat = 24
-    lazy var breachAlertImageView: UIImageView = {
-        let imageView = UIImageView(image: BreachAlertsManager.icon)
-        imageView.isHidden = true
-        return imageView
-    }()
-    lazy var breachAlertContainer: UIView = {
-        let view = UIView()
-        view.addSubview(breachAlertImageView)
+    lazy var breachAlertImageView: UIImageView = .build { imageView in
+        imageView.image = BreachAlertsManager.icon
+    }
+
+    lazy var breachAlertContainer: UIView = .build { [weak self] view in
+        guard let self = self else { return }
+
+        view.addSubview(self.breachAlertImageView)
         view.setContentHuggingPriority(.defaultHigh, for: .horizontal)
-        return view
-    }()
+    }
+
     lazy var breachMargin: CGFloat = {
-        return breachAlertSize+LoginTableViewCellUX.HorizontalMargin*2
+        return breachAlertSize + LoginTableViewCellUX.HorizontalMargin * 2
     }()
-    
-    lazy var hostnameLabel: UILabel = {
-        let label = UILabel()
+
+    lazy var hostnameLabel: UILabel = .build { label in
```

Comment:
At this point, does it really make sense to use `.build`? There weren't any runtime issues before and it feels smelly having to work around it in this way. There are plenty of stack views within the codebase that don't follow this pattern. Here's what I had before:

```swift
private lazy var textStack: UIStackView = {
        let stack = UIStackView(arrangedSubviews: [hostnameLabel, usernameLabel])
        stack.translatesAutoresizingMaskIntoConstraints = false
        stack.axis = .vertical
        stack.isLayoutMarginsRelativeArrangement = true
        stack.spacing = 8
        stack.layoutMargins = .init(top: 8, left: 0, bottom: 8, right: 0)
        return stack
    }()
```

Reply:
I gave this a try and the runtime issues persisted ^ 

Here's what fixed it

```swift
private lazy var textStack: UIStackView = .build { [weak self] stack in
    guard let self = self else { return }

    stack.addArrangedSubview(self.hostnameLabel)
    stack.addArrangedSubview(self.usernameLabel)
    stack.axis = .vertical
    stack.alignment = .fill
    stack.distribution = .fillProportionally
    stack.isLayoutMarginsRelativeArrangement = true
    stack.spacing = 8
    stack.layoutMargins = .init(top: 8, left: 0, bottom: 8, right: 0)
}
```

Reply:
Strange, I had tried it on my end and it was ok. But setting distribution is much better anyway, thanks :)

Reply:
Just to give a bit more context, here's a great article about compression resistance and content hugging: https://abhimuralidharan.medium.com/ios-content-hugging-and-content-compression-resistance-priorities-476fb5828ef

Hope this can help!